app/boards/shields/helix/README.md

```diff
@@ -0,0 +1,10 @@
+Note to user:
+
+(1)	Peripheral RGB function is impaired until full support is implemented in the master branch.
+(2)	OLED displays are not currently included in this shield. This will be updated after OLED support is live.
+(3)	Definitions for 'KANA' and 'EISUU' input are currently utilized under 'LANG1' and 'LANG2' respectively.
+
+Thanks to NiceII, KemoNine, PeteJohanson, TJ "Chormbo The Great", Joelspadin/Rinh, Wofiel, Okke,
```

Comment:
Nitpick, Nicell (LL) not NiceII (ii) 😄

Reply:
@Nicell I appreciate you taking the time to review my initial submission for the Helix shield. As requested, I made the following changes:
**1.** Adjusted all .conf files to reflect instructed reductions.
**2.** Included Helix shield definitions in setup files.
**3.** Made grammar/syntax adjustments and eliminated typos.
With regards to make two commits, I noticed a small error in the _setup.ps1_ file after committing and pushing changes. That should now be rectified in the third commit and push. I am still learning about the Git process, but strive to do it effectively. Please instruct me if this is unsatisfactory, or if there are any other errors remaining. Thank you.